Kernel/Net/Ethernet.h

```diff
@@ -8,8 +8,17 @@
 
 #include <AK/Endian.h>
 #include <AK/MACAddress.h>
+#include <AK/Types.h>
 
-#pragma GCC diagnostic ignored "-Warray-bounds"
```

Comment:
is there a reason for removing this line? As in, did this throw a warning before and it doesn't anymore..?

Reply:
I think It was resolved after adding packed attribute

Reply:
That's incorrect. The pragma was introduced with ac215ca601393dc111643b119835ea39e6c9bd23, while the packed struct was already there. 

It's a very old commit, so it's not unlikely that it's indeed not needed anymore - but removing it blindly is something I'm not happy about.

Reply:
Alright then, please make a separate commit. Such changes are always welcome.